camunda-scaling-operator/api/v1alpha1/zeebeautoscaler_types.go

```diff
@@ -43,10 +43,14 @@ type ZeebeAutoscalerStatus struct {
 	// Conditions holds the information on the last operations on Zeebe that can be useful during scaling
 	// +kubebuilder:validation:Optional
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+
+	Replicas int32  `json:"replicas"`
+	Selector string `json:"selector"`
 }
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
```

Comment:
FYI @npepinpe so that you can grok it. 

This marker from [kubebuilder](http://kubebuilder.io) is used to create the OpenAPI spec (file in crd/bases/camunda....).